templates/bot-msgext/js/default/package.json

```diff
@@ -13,7 +13,7 @@
     "watch": "nodemon ./index.js"
   },
   "dependencies": {
-    "teamsdev-client": "^1.0.0",
+    "@microsoft/teamsfx": "0.1.0-alpha.3",
```

Comment:
Is it possible to use something like ^0.1.0? or why choosing a fixed version?

Reply:
Offline discussed